docker/install-shared-deps.sh

```diff
@@ -1,19 +1,63 @@
 #!/usr/bin/env bash
 # Script for shared dependencies
 
-set -o errexit -o pipefail -o nounset
+set -x -o errexit -o pipefail -o nounset
 
-apt-get update -y && apt-get install -y software-properties-common
-add-apt-repository -y ppa:valhalla-core/valhalla && apt-get update -y
 
-apt-get install -y \
+# Lets start with setting us up for installing the non-trusted ppa of primeserver
+apt-get update --assume-yes
+apt-get install --assume-yes \
+    gnupg2
+
+# We're manually adding the sources since the ppa has no 20.04 package yet
+cat << EOF > /etc/apt/sources.list.d/foo.list
+deb http://ppa.launchpad.net/valhalla-core/valhalla/ubuntu bionic main
+deb-src http://ppa.launchpad.net/valhalla-core/valhalla/ubuntu bionic main
```

Comment:
yeah im blocking on this. i have prime_server ready to go on 20.04 the problem is i just need to push it up when i find the time and let the ppa build it. ill do that when i get a free moment and then we can simplify this